Formula/f/fdroidserver.rb

```diff
@@ -385,10 +398,20 @@ def caveats
       (testpath/"fdroid/config/categories.yml").write <<~YAML
         Development:
           name: Development
+          icon: category_development.png
         System:
           name: System
+          icon: category_system.png
       YAML
 
+      # fake PNGs, they just start with PNG file magic bytes
+      (testpath/"fdroid/config/category_development.png").write <<~PNG
+        \x89\x50\x4E\x47\x0D\x0A\x1A\x0A\x00\x00\x00
```

Comment:
This is causing `An error occurred while FormulaAudit/PythonVersions cop was inspecting /opt/homebrew/Library/Taps/homebrew/homebrew-core/Formula/f/fdroidserver.rb:1:0.`

Any ideas @Homebrew/core?

Reply:
Not sure on error, but if a fake PNG is needed then can use `test_fixtures("test.png")` which uses 
https://github.com/Homebrew/brew/blob/master/Library/Homebrew/test/support/fixtures/test.png

Reply:
I've updated the test to use this instead. I've also run `brew update-python-resources` which added a few more that were not previously listed as resources.